account_payment_confirm/tests/test_account_payment_confirm.py

```diff
@@ -0,0 +1,35 @@
+import time
+
+from odoo.exceptions import ValidationError
+
+from .common import ImprovePaymentConfirmationTest
```

Comment:
Why having 2 files for tests, when you can have only one? You end up inheriting this class anyway
is there a real reason? :thinking:

Reply:
@ivantodorovich : I keep static variables in setup and inherit it

Reply:
But you can do it like this: https://github.com/druidoo/FoodCoops/blob/12.0/coop_purchase/tests/test_coop_purchase.py

There's no need to inherit it :thinking: